src/plugins/data/public/plugin.ts

```diff
@@ -220,6 +228,9 @@ export class DataPublicPlugin
         notifications.toasts,
         application.navigateToApp
       ),
+      isCurrentWorkspaceOwner:
+        workspaces?.currentWorkspace$.getValue()?.owner ||
+        application.capabilities?.dashboards?.isDashboardAdmin !== false,
```

Comment:
```suggestion
      isCurrentWorkspaceOwner:
        !!application.capabilities?.workspace?.enabled &&
        (
          workspaces?.currentWorkspace$.getValue()?.owner ||
          application.capabilities?.dashboards?.isDashboardAdmin !== false
        )
```
Can we explicitly do a detect here to see if workspace is enabled?

Reply:
Sure, updated